timetracker/entries/views.py

```diff
@@ -68,23 +63,22 @@ def entries(request):
     })
 
 
-def projects(request):
-    if request.method == 'POST':
-        # Create our form object with our POST data
-        form = ProjectForm(request.POST)
-        if form.is_valid():
-            Project.objects.create(
-                name=form.cleaned_data['name'],
-                client=form.cleaned_data['client']
-            )
-    else:
-        form = ProjectForm()
+class ProjectCreateView(CreateView):
+    """
+    View to create a Project instance every
+    time the form is submitted
+    """
+    project = Project.objects.get(pk=1)
+    model = Project
+    template_name = 'projects.html'
+    form_class = ProjectForm
+    success_url = '/projects/'
+    initial = {'name':project.name}
```

Comment:
... neither this line, as they are executed the first time this python file is loaded and won't change afterwards.